pkg/pillar/cmd/zedrouter/probe.go

```diff
@@ -336,7 +330,9 @@ func launchHostProbe(ctx *zedrouterContext) {
 				info.IfName, info.NhAddr, info.GatewayUP, info.RemoteHostUP)
 
 			// Local nexthop ping, only apply to Ethernet type of interface
-			if info.IsFree {
+			// XXX should we probe other priorities if they are
+			// in use?
+			if info.FallbackPriority == 0 {
```

Comment:
@naiming-zededa what about this question?

Reply:
this is the 'local ping' to the next hop, every 15 seconds, vs to the remote probe being 2.5 min. So if it's a paid service, we don't want the traffic. so keep this in prio 0 is fine.